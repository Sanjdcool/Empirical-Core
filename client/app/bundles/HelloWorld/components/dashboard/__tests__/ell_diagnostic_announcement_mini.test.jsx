import React from 'react';
import { shallow } from 'enzyme';

import EllDiagnosticAnnouncementMini from '../ell_diagnostic_announcement_mini';

describe('EllDiagnosticAnnouncementMini component', () => {

  it('should render', () => {
    expect(shallow(<EllDiagnosticAnnouncementMini />)).toMatchSnapshot();
  });

});
