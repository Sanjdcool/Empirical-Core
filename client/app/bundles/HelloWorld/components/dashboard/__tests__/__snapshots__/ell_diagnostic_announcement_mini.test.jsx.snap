// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EllDiagnosticAnnouncementMini component should render 1`] = `
<div
  className="mini_container results-overview-mini-container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="gray-underline"
      style={
        Object {
          "padding": "8px",
          "position": "relative",
        }
      }
    >
      <img
        src="/assets/icons/icon-star.svg"
        style={
          Object {
            "height": "27px",
            "position": "absolute",
            "right": "-22px",
            "top": "-11px",
            "transform": "rotate(-22deg)",
            "width": "27px",
          }
        }
      />
      <img
        src="/assets/icons/icon-star.svg"
        style={
          Object {
            "height": "14px",
            "position": "absolute",
            "right": "-3px",
            "top": "-11px",
            "transform": "rotate(-34deg)",
            "width": "14px",
          }
        }
      />
      <h3
        style={
          Object {
            "fontWeight": "normal",
          }
        }
      >
        New ELL Diagnostic
      </h3>
    </div>
    <div>
      <a
        href="/tools/diagnostic"
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "center",
            "marginTop": "8px",
          }
        }
      >
        <img
          src="/images/icon-diagnostic-black.svg"
          style={
            Object {
              "height": "25px",
              "marginRight": "10px",
              "marginTop": "3px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "18px",
              "fontWeight": "bold",
              "margin": "8px 0 0",
            }
          }
        >
          ELL Diagnostic
        </p>
      </a>
    </div>
    <p
      style={
        Object {
          "fontFamily": "lucida-grande, adelle-sans, helvetica",
          "lineHeight": "1.57",
          "marginTop": "16px",
          "padding": "0px 15px",
        }
      }
    >
      Our new ELL Diagnostic assesses your studentsâ€™ knowledge of 10 different concepts specifically chosen for English language learners.
    </p>
    <a
      href="/tools/diagnostic"
    >
      <button
        className="button button-white beta"
        style={
          Object {
            "fontWeight": "normal",
            "maxWidth": "233px",
            "paddingBottom": "13px",
            "width": "100%",
          }
        }
      >
        Learn More
      </button>
    </a>
  </div>
</div>
`;
